drivers/soundwire/generic_bandwidth_allocation.c

```diff
@@ -206,6 +206,13 @@ static void _sdw_compute_port_params(struct sdw_bus *bus,
 			port_bo = 1;
 
 			list_for_each_entry(m_rt, &bus->m_rt_list, bus_node) {
+				/*
+				 * Only runtimes with CONFIGURED, PREPARED, ENABLED, and DISABLED
+				 * states should be included in the bandwidth calculation.
+				 */
+				if (m_rt->stream->state > SDW_STREAM_DISABLED ||
+				    m_rt->stream->state < SDW_STREAM_CONFIGURED)
+					continue;
```

Comment:
commit message: s/we should not take in count/we should not take into account/
?

Reply:
> commit message: s/we should not take in count/we should not take into account/ ?

fixed